ros/src/util/packages/data_preprocessor/scripts/top_urg.launch

```diff
@@ -1,5 +1,30 @@
 <launch>
-  <param name="hokuyo_node/calibrate_time" type="bool" value="false" />
-  <param name="hokuyo_node/port" type="str" value="/dev/ttyACM0" />
-  <node name="hokuyo_node" pkg="hokuyo_node" type="hokuyo_node" />
-</launch>
+
+<!--  When using an IP-connected LIDAR, populate the "ip_address" parameter with the address of the LIDAR.
+      Otherwise, leave it blank. If supported by your LIDAR, you may enable the publish_intensity
+      and/or publish_multiecho options. -->
+
+
```

Comment:
Maybe add this comment:    
`<!--  Note: when using a USB interface device please make sure the ip_address parameter is the empty string, otherwise the driver will choose the Ethernet interface. -->`